tests/Integration/Behaviour/Features/Context/Domain/ManufacturerFeatureContext.php

```diff
@@ -290,4 +312,24 @@ private function createManufacturerUsingCommand($reference, array $data)
 
         SharedStorage::getStorage()->set($reference, new Manufacturer($manufacturerId->getValue()));
     }
+
+    /**
+     * @Then manufacturer :manufacturerReference should have added addresses
+     *
+     * @param string $manufacturerReference
+     */
+    public function manufacturerShouldHaveFollowingProperties(string $manufacturerReference)
+    {
+        /** @var Manufacturer $manufacturer */
+        $manufacturer = SharedStorage::getStorage()->get($manufacturerReference);
+        /** @var ViewableManufacturer $viewableMaufacturer */
+        $viewableMaufacturer = $this->getQueryBus()->handle(new GetManufacturerForViewing(
+            $manufacturer->id,
+            (int) $this->getContainer()->get('prestashop.adapter.legacy.configuration')->get('PS_LANG_DEFAULT')
+        ));
+
+        Assert::assertSame($manufacturer->name, $viewableMaufacturer->getName());
+        Assert::assertSame(2, count($viewableMaufacturer->getManufacturerAddresses()));
+        Assert::assertSame(0, count($viewableMaufacturer->getManufacturerProducts()));
```

Comment:
Maybe thoses hardcoded values in assertions should be passed by parameter and specified in the scenario? So that this method can be used with different values ?